src/Host/Broker/Impl/NativeMethods.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.R.Host.Broker {
```

Comment:
Most of definitions here are unused (or rather used by `ProcessHelpers`, which is unused - see comment there). We will likely need to switch from `Process` to direct Win32 calls for security purposes, though, and then it'll come in handy.